pandas/io/parsers.py

```diff
@@ -1685,7 +1685,8 @@ def _cast_types(self, values, cast_type, column):
 
         else:
             try:
-                values = astype_nansafe(values, cast_type, copy=True)
+                values = astype_nansafe(values, cast_type,
```

Comment:
@gfyoung this is a longshot but any idea why you passed skipna=True here but not 9 lines up?